server/config/default.json

```diff
@@ -23,6 +23,7 @@
   "SENDER_EMAIL": "contact@tomochain.com",
 
   "CLIENT_URL": "http://localhost:3000/",
+  "MASTERNODE_URL": "http://localhost:3001/",
```

Comment:
Change param name to "TOMOMASTER_URL"
Also need to add to custom-environment-variables.json